vep-mapping-pipeline/vep_mapping_pipeline/test/output_mappings.tsv

```diff
@@ -1988,7 +1987,7 @@ X:2828984:G:C	1	ENSG00000056998	GYG2	intron_variant	0
 X:28587759:GT:G	1	ENSG00000169306	IL1RAPL1	5_prime_UTR_variant	0
 X:31178783:C:G	1	ENSG00000198947	DMD	missense_variant	0
 X:31204120:T:C	1	ENSG00000198947	DMD	splice_acceptor_variant	0
-X:31508251:C:T	1	ENSG00000198947	DMD	5_prime_UTR_variant	0
+X:31508251:C:T	1	ENSG00000198947	DMD	missense_variant	0
```

Comment:
@M-casado @apriltuesday Worth looking at in the genome browser. Is there indeed an exon (in any transcript) at this location to justify the `missense_variant` consequence?

Reply:
In this case I can see at those coordinates one of DMD's transcript variants (_DMD-205_) **does have a coding exon**. These coordinates are close to its 5' UTR, nevertheless, which could easily lead to a 5' UTR in some cases. Besides, there are possible stopping codon right behind this variant, which again supports a 5' UTR variant. 

In summary, although the change does not seem technically incorrect, if you believe there should be further investigations regarding it, I would need to take a look at this specific transcript variant to check: **(1)** if there have been reports of alternative splicing of its 5' UTR; and **(2)** check its reading frame to account for the possible stopping codons and splicing donors/targets supporting its previous status (5' UTR variant).  
![image](https://user-images.githubusercontent.com/77288380/114363239-7ec2ab80-9b78-11eb-9e8b-8d4c2619c8a5.png)